### DISCUSSION TITLE
The command does not exist

### DISCUSSION THREAD

**Discussion by OctolingNexus** [getpelican/pelican#3175] [Q&A]
pelican and its sub commands does not exist when installed any fixes? its there inthe files but cmd is saying nope

**Comment by justinmayer** [comment]
There isn't nearly enough info here to even begin to provide any guidance. Please start here and then update your description with more information: https://docs.getpelican.com/en/latest/contribute.html#how-to-get-help

If you are using Windows, I suggest trying [Pipx](https://pypa.github.io/pipx/installation/):

```
py -3 -m pip install --user pipx
py -3 -m pipx ensurepath
py -3 -m pipx install "pelican[markdown]"
```

**Comment by OctolingNexus** [comment]
wdym not enough info the info is literally right there

the command doesnt exist thats what python is saying

how is there not enough info? because thats it

**Reply by justinmayer** [reply]
What operating system and version? Which Python interpreter did you use to install Pelican? What was the exact command you used to install Pelican?

I can think of a dozen other questions, the answers to which should have been provided in the original question without being asked. Maintainers can barely find the time to work on the project, much less help folks who won't take the time to communicate in a way that engenders that help.

**Reply by OctolingNexus** [reply]
latest version of python and i used the commands that was used to install was py -m pip install "pelican[markdown]"
and i used cmd to run the command

**Reply by justinmayer** [reply]
You still have not provided your OS and version. I suspect you are using Windows, which Pelican maintainers do not use and have next-to-zero experience with.

**Reply by OctolingNexus** [reply]
oh i forgot to put it im using windows yea

**Reply by OctolingNexus** [reply]
windows 10 to be more specific

**Reply by justinmayer** [reply]
Updated my original answer with a suggestion. Try that and report back with the results.

**Reply by avaris** [reply]
Run `pelican` via `py -m pelican [args]` or
Make sure you added your default python to `PATH` during installation (see the bottom option in the [screenshot](https://docs.python.org/3/using/windows.html#installation-steps)) or
Find out your python installation folder and `pelican` executable within, should be under `Scripts` folder for that python, and add that to windows `PATH` [manually](https://docs.python.org/3/using/windows.html#excursus-setting-environment-variables).
